raw-video/RawVideo/ContentView.swift

```diff
@@ -125,18 +134,18 @@ struct ContentView : View
                     {
                         VStack {
                             TextEditor(text: $token)
-                            .frame(width: 250, height: 30)
-                            .padding(10)
-                            .overlay(RoundedRectangle(cornerRadius: 5)
-                                .stroke(Color.black, lineWidth: 2))
-                            .focused($isFocused)
-                            .onChange(of: token) { _ in
-                                if token.last?.isNewline == .some(true)
-                                {
-                                    token.removeLast()
-                                    isFocused = false
+                                .frame(width: 250, height: 30)
+                                .padding(10)
+                                .overlay(RoundedRectangle(cornerRadius: 5)
+                                    .stroke(Color.black, lineWidth: 2))
+                                .focused($isFocused)
+                                .onChange(of: token) { _ in
+                                    if token.last?.isNewline == .some(true)
```

Comment:
```suggestion
                                    if token.last?.isNewline == true
```

Reply:
Yes, optionals in swift can either is an enumeration so it can be either .some(value) or .none (aka nil), but we don't need to use `.some` explicitly, `token.last?.isNewline == true` is fine.